LathBotFront/Commands/ReactionCommands.cs

```diff
@@ -100,6 +100,8 @@ public async Task Pat(CommandContext ctx, [Description("Who to pat")]DiscordMemb
                 await ctx.RespondAsync(new DiscordMessageBuilder().WithSticker(ctx.Guild.Stickers[964256150054899742]));
             else if (member.Id == 700373370491109489)//femke
                 await ctx.RespondAsync(new DiscordMessageBuilder().WithSticker(ctx.Guild.Stickers[1013888253272801310]));
+            else if (member.id == 387325006176059394)//julian
+                await ctx.RespondAsync($"Julian!, you just got pat by {ctx.Member.Mention")
```

Comment:
```suggestion
                await ctx.RespondAsync($"Julian!, you just got pat by {ctx.Member.Mention");
```
semicolon missing here